features/home_page/maintain_org_pos_in_list.feature

```diff
@@ -4,11 +4,10 @@ Feature: maintain position in organisation list
   I would like to open each organisation page in new tab/window
   
   Background: organisations have been added to database
-    
     Given the following organisations exist:
       | name                           | description                    | address        | postcode | website       |
       | Harrow Bereavement Counselling | Harrow Bereavement Counselling | 34 Pinner Road | HA1 4HZ  | http://a.com/ |
 
   Scenario:
     When I visit the organisations index page
-    Then I should see link "Harrow Bereavement Counselling" targeting new page
+    Then I should see link "Harrow Bereavement Counselling" targeting new page
```

Comment:
I noticed that you sometimes add return line to end of statements. Maybe you could configure your editor to avoid this case :)